modules/stochastic_tools/include/state_sim/model/StateSimBase.h

```diff
@@ -0,0 +1,123 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef STATESIMBASE_H
+#define STATESIMBASE_H
+
+#include "MooseRandom.h"
+#include <string>
+#include <stdexcept>
+
+#define UNUSED(expr) do { (void)(expr); } while (0)
```

Comment:
Get rid of this. The way to get rid of the compiler error for unused variables is to comment out the formal parameter name (i.e. just leave the type where the parameter is unused in a method).